navigator/src/main/frontend/src/components/Navigator.vue

```diff
@@ -43,7 +43,10 @@
     <b-table bordered :items="items" :fields="fields" :filter="filter" class="text-left">
       <template slot="label" scope="label" >
         <span v-if="label.item.type === 'entity'">
-            <a :href="'/menu/main/dataexplorer?entity=' + label.item.id" target="_blank">
+            <span v-if="label.item.abstract">
+              {{label.item.label}} (abstract)
```

Comment:
Lets ask @mswertz, but I think abstract entities should not be shown at all, the goal of this plugin is to navigate your actual data.